### PR TITLE
Refactors _quickicons.less

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
Refactors the  admin templates _quickicons.less

### Testing Instructions
Apply PR and ensure the quickicons display correctly. No changes in styling so they should appear as before.

image

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 23486bdab78578ea847aca336d3bb7ed977957e5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18278">issues.joomla.org/tracker/joomla-cms/18278</a>.</sub>

File: administrator/templates/isis/less/blocks/_quickicons.less [hunk 1]
```diff
@@ -1,47 +1,27 @@
...
-	line-height: 18px;
-}
-
-.quick-icons .nav-list [class^="icon-"], .quick-icons .nav-list [class*=" icon-"] {
-	margin-right: 9px;
+	.nav-header {
+		margin: 12px 0 5px;
+		font-size: 13px;
+		&:first-child {
+			margin: 0px 0 5px;
```
Comment (Quy) [inline administrator/templates/isis/less/blocks/_quickicons.less]: Remove px from `0px`

**Comment by ciar4n** [comment]
@Quy Ty... amended.

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on 002b90b97b82536bdc41c14a3074d89aa1537f22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18278">issues.joomla.org/tracker/joomla-cms/18278</a>.</sub>

**Comment by dgrammatiko** [comment]
RTC

**Pull request merged**